docs/reference/changelog-r2020.md

```diff
@@ -13,19 +17,34 @@ Released on XXX YYth, 2019.
       - Introduced the `Background.*IrradianceUrl` fields to define an `HDR` irradiance map.
       - Added image tools to help with `HDR` format and equirectangular projections.
       - Added new HDR background: `entrance_hall`
-    - Added new appearance: `DryMud`
+    - Added several new appearances: `CorrugatedPlates`, `CorrugatedPvc`, `FormedConcrete`, and `DryMud`.
+    - Replaced the [Viewpoint](viewpoint.md) `followOrientation` field by a `followType` field for more flexibility.
     - Improved the Webots online 3D viewer: `webots.min.js`
       - Improved support of the Webots rendering pipeline: supported the Bloom post-processing effect.
       - Added support for the `ImageTexture.filtering` field.
       - Improved the console log history. Added a button to clear the console.
     - Improved [robotbenchmark](https://robotbenchmark.net) worlds.
       - Improved overall graphics quality (using the PBR materials and the HDR backgrounds).
       - Improved `humanoid_sprint` benchmark metrics.
-    - Replaced the [Viewpoint](viewpoint.md) `followOrientation` field by a `followType` field for more flexibility.
+    - Added a [script to cleanup the Webots preferences](https://github.com/cyberbotics/webots/blob/master/scripts/preferences_cleaner/README.md).
+    - Linux: Added support for Python 3.8.
   - Enhancements
     - Improved the [Sick LD MRS](../guide/lidar-sensors.md#sick-ld-mrs) PROTO to support the following types: `400001`, `400102`, `400001S01`, `400102S01` and `800001S01`.
-    - Split the Webots and controller libraries to avoid possible conflicts with external libraries.
     - Set the [ABB IRB 4600/40](../guide/irb4600-40.md) root node to [Robot](robot.md) instead of [Solid](solid.md) to be able to insert it everywhere.
+    - Webots now waits for extern controllers if the `Robot.synchronization` field is set to `TRUE`.
+    - Device names are displayed in the scene tree next to the node name.
+    - Split the Webots and controller libraries to avoid possible conflicts with external libraries.
     - Windows/Linux: Move the `Check for updates...` menu from `Tools` to `Help` for consistency with other applications.
-  - New Samples
-    - Added a `complete_apartment` world.
+  - Bug fixes
+    - Fixed [Lidar](lidar.md) point cloud access in controllers (thanks to Alexander).
```

Comment:
```suggestion
    - Fixed the [`wb_supervisor_node_reset_physics()`](supervisor.md#wb_supervisor_node_reset_physics) function when applied on dynamic objects (in motion and containing joints).
    - Fixed [Lidar](lidar.md) point cloud access in controllers (thanks to Alexander).
```

Reply:
Thank you. Probably the branch was merged after I created this change.